src/utils/test.rs

```diff
@@ -47,15 +49,17 @@ fn connected_strat<G: PortView + LinkView + PortMut + fmt::Debug>(
     })
 }
 
-pub(crate) fn gen_portgraph_connected(
+/// Proptest strategy for generating a connected portgraph.
```

Comment:
That's true, it shouldn't be here. 

On a side note: it's made public so that I can access it from integration tests, not sure if this is bad style...

Either way I will remove it from this PR.